native/src/components/PoiDetailRow.tsx

```diff
@@ -4,17 +4,14 @@ import styled from 'styled-components/native'
 
 import openExternalUrl from '../utils/openExternalUrl'
 
-const Row = styled.View`
-  flex-direction: row;
-  padding-vertical: 3px;
-`
-
 const Icon = styled.Image`
   margin-horizontal: 5px;
 `
 
 const StyledPressable = styled.Pressable`
-  align-self: center;
+  align-self: flex-start;
```

Comment:
Not tested if this works as well but you could remove this and add `align-self: center` as style to the Text component.